api/tests/routes/backoffice/finance_test.py

```diff
@@ -13,6 +13,8 @@
 from pcapi.core.finance import utils as finance_utils
 from pcapi.core.history import factories as history_factories
 from pcapi.core.history import models as history_models
+import pcapi.core.mails.testing as mails_testing
```

Comment:
Pour l'homogénéité du code du backoffice : 
```suggestion
from pcapi.core.mails import testing as mails_testing
```